src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,69 @@
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 500px;
+  height: 500px;
+  background-color: aqua;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+
+  &__hands {
+    width: 400px;
+    height: 400px;
+    position: relative;
+
+    &-minutes {
+      position: absolute;
+      left: 50%;
+      top: 30px;
+      width: 5px;
+      height: 170px;
+      box-sizing: border-box;
+      background-color: rgb(0, 174, 255);
+      animation: rotate steps(60) var(--hour) infinite;
+      transform-origin: center bottom;
+    }
+
+    &-seconds {
+      position: absolute;
+      left: calc(50% + 1.5px);
+      width: 2px;
+      height: 200px;
+      background-color: rgb(255, 0, 0);
+      animation: rotate steps(60) var(--minute) infinite;
+      transform-origin: center bottom;
+
+      @keyframes rotate {
+        from {
+          transform: rotate(0);
+        }
+
+        to {
+          transform: rotate(1turn);
+        }
+      }
+    }
+  }
+
+  &__circle {
+    width: 20px;
+    height: 20px;
+    background-color: yellow;
+    border-radius: 50%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-40%, -50%);
```

Comment:
I don't understand why you use 40% but not 50%
It is not clear how this number is calculated

Reply:
Yep, my logic failed.
I should have checked in the Dev Tools which elements are properly center before doing any calculations.

In my mind the circle was off and the hands were centered, but it was vice versa.

I really appreciate you pointing me where I'm failing.

Thank you!